tests/dummy/app/pods/expand/template.hbs

```diff
@@ -0,0 +1,245 @@
+          hook='myExpand'
+          expandLabel='View information'
+          collapseLabel='Hide information'
+        }}
+          Detailed information.
+        {{/frost-expand}}
+        {{! END-SNIPPET expand-set-labels}}
+      </div>
+      <br/>
+      <div class="snippet">
+        {{code-snippet name="expand-set-labels.hbs"}}
+      </div>
+    </div>
+  </div>
+
+  <div class='section-title'>Set Duration</div>
+  <hr>
+  <div class="section">
+    <div class="example">
+      <div class="title">default</div>
+      <div class="demo">
+        {{! BEGIN-SNIPPET expand-default-duration}}
+        {{#frost-expand
+          hook='myExpand'
+        }}
+          Expanded content.
+        {{/frost-expand}}
+        {{! END-SNIPPET expand-default-duration}}
+      </div>
+      <br/>
+      <div class="snippet">
+        {{code-snippet name="expand-default-duration.hbs"}}
+      </div>
+    </div>
+
+    <div class="example">
+      <div class="title">slow</div>
+      <div class="demo">
+        {{! BEGIN-SNIPPET expand-slow-duration}}
+        {{#frost-expand
+          hook='myExpand'
+          duration='slow'
+        }}
+          Expanded content.
+        {{/frost-expand}}
+        {{! END-SNIPPET expand-slow-duration}}
+      </div>
+      <br/>
+      <div class="snippet">
+        {{code-snippet name="expand-slow-duration.hbs"}}
+      </div>
+    </div>
+
+    <div class="example">
+      <div class="title">fast</div>
+      <div class="demo">
+        {{! BEGIN-SNIPPET expand-fast-duration}}
+        {{#frost-expand
+          hook='myExpand'
+          duration='fast'
+        }}
+          Expanded content.
+        {{/frost-expand}}
+        {{! END-SNIPPET expand-fast-duration}}
+      </div>
+      <br/>
+      <div class="snippet">
+        {{code-snippet name="expand-fast-duration.hbs"}}
+      </div>
+    </div>
+
+    <div class="example">
+      <div class="title">none</div>
+      <div class="demo">
+        {{! BEGIN-SNIPPET expand-none-duration}}
+        {{#frost-expand
+          hook='myExpand'
+          duration='none'
+        }}
+          Expanded content.
+        {{/frost-expand}}
+        {{! END-SNIPPET expand-none-duration}}
+      </div>
+      <br/>
+      <div class="snippet">
+        {{code-snippet name="expand-none-duration.hbs"}}
+      </div>
+    </div>
+  </div>
+
+  <div class='section-title'>Events</div>
+  <hr>
+  <div class="section">
+    <div class="example">
+      <div class="demo">
+        {{! BEGIN-SNIPPET expand-events}}
+        {{#frost-expand
+          hook='myExpand'
+          onExpand=(action 'onExpandHandler')
+          onCollapse=(action 'onCollapseHandler')
+        }}
+          Expanded content.
+        {{/frost-expand}}
+        {{! END-SNIPPET expand-events}}
+      </div>
+      <br/>
+      <div class="snippet">
+        {{code-snippet name="expand-events.hbs"}}
+      </div>
+    </div>
+  </div>
+
+  <div class='section-title'>Query Parameter</div>
```

Comment:
Don't worry about providing a specific QP example - this is an option available to any case where external state is held and consumers should be familiar with this as an Ember option